### PR TITLE
Fix lazy plotter

### PR DESCRIPTION
**Pull request by mberr**

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -145,14 +145,20 @@ def train(
         num_epochs=num_epochs,
         batch_size=256,
         callbacks=[
-            EntityPlotCallback(
```
Comment (mberr) [inline cli.py]: This change is not related to the PR :sweat_smile:

**Comment by cthoyt** [comment]
Thanks @mberr. I combine your new implementation to be the standard and also made the static axes optional
